index.html

```diff
@@ -51,20 +65,30 @@ <h1 class="project-heading">Projects</h1>
     <!--Grid-container for projects-->
 
     <section class="grid-container">
+        
+        <!--Project heading for desktop-->
+
+        <h1 class="desktop-heading">Projects</h1>
+        <img class="more-dots" alt="dots" src="./images/dots.png">
+        <img class="semicircle" alt="shapes" src="./images/semicircle.png">
+        <img class="two-squares" alt="shapes" src="./images/twosquares.png">
+        <img class="diamond" alt="shape" src="./images/diamond.png">
+        <img class="shapes" alt="shapes" src="./images/shapes.png">
+        <img class="yellow-tear" alt="shape" src="./images/yelllowtear.png">
 
         <!--Project 1-->
 
         <section class="project project1">
```

Comment:
Please try to match the Figma design in width 993px for the projects and about sections.
| Your design | Figma design |
| --- | --- |
| ![screencapture-raw-githack-emyrue-Portfolio-form-index-html-2022-02-17-01_25_45](https://user-images.githubusercontent.com/24703171/154375289-29eff66a-70ab-4488-91c7-fcc42d7ccc0b.png) | ![Template 8-main](https://user-images.githubusercontent.com/24703171/154375261-638a6ffd-f8a7-4f12-97a7-bafcbea2a853.png) |

And in width 992px the image that takes class `banner2` disappear and this is how the page looks like:
![screencapture-raw-githack-emyrue-Portfolio-form-index-html-2022-02-17-01_38_25](https://user-images.githubusercontent.com/24703171/154375896-4b6da155-5a4c-4cb0-9eb1-2def5580114f.png)